manifests/windows/install.pp

```diff
@@ -56,4 +56,11 @@
     command => "${::system32}\\cmd.exe /c start /b ${_cmd_location} /c \"${_installbat}\" ${::puppet_agent_pid}",
     path    => $::path,
   }
+
+  # PE-15037 Cache dir loses inheritable SYSTEM perms
```

Comment:
I would also note the original PUP ticket here for ref - https://tickets.puppetlabs.com/browse/PUP-5480